.github/ISSUE_TEMPLATE/stable_feature.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Stable Feature Requirements
+about: Requirements for promoting a feature to stable
+---
+
+This page lists the requirements for promoting a feature to stable. Promotion to stable includes all requirements of promotion to beta. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+
+**Promotion to Alpha:**
+
+**Promotion to Beta:**
+
+**Relevant Documentation:**
+
+```
+```
+
+--- 
+
+### Requirements: 
+
+**Tests**
+
+- [ ] Automated tests are in place to prevent regressions. 
+
+**Performance**
+
+- [ ] Latency, throughput, and scalability are quantified and documented on
+	istio.io. 
+
+**Bugs**
+
+- [ ] Feature has no known major issues. 
```

Comment:
What's a major issue?

Reply:
Whatever the owning work group decides it is. Say crashes, feature not working, feature not working in certain scenarios... this used to be called P0/P1, but there's no such thing as a P1/P0 bug in terms of a feature (those are release level issues).